Articles/Blog/HelloHeatEquation.md

```diff
@@ -1,143 +1,99 @@
-
-While this code is almost identical the hand-coded example above, this library does include some extra features that allows users to implement some additional functionality including writing results to different kinds of R8 files, adding a timestamp, and solving for the Courant-Friedrichs-Loewy coefficient.
-
-### More Implementations!
-Using the Heat Equation to demonstrate the power of a programming model or mathematical library is not new. In fact, it is common tutorial exercise for new scientific programmers. Since there are too many interesting examples to include in this article, we've started [a repository showcase](https://github.com/betterscientificsoftware/hello-heat-equation) to collect the many examples.
+## Why this form of the Heat Equation?
+While the one dimensional heat equation might not be sufficient to
+model solar heating of the Earth, even in this relatively simple form,
+interesting and meaningful science questions may be investigated involving
+both transient and steady-state behavior. Examples are determining if water
+pipes embedded in log cabin (solid wood) walls would freeze during a cold
+storm or the required thickness of insulation around fuel pipeline to avoid
+combustion from a fast burning brush fire. These and many other questions
+involving heat flow can be investigated with this relatively simply form of
+the heat equation.
+
+Solutions to the one dimensional heat equation are also very easy for
+beginners to visualize. This can help them understand how different
+approaches work. The simplicity of the physics and mathematics also
+allows beginners to focus more on the implementation details; the
+programming language, parallel execution paradigm, discretization and
+mathematical support libraries.
+
+## A Hello CSE World Repository
+We are announcing the beginnings of an effort to populate and maintain a
+repository of implementations of the heat equation demonstrating various
+aspects of CSE software. The resulting repository will be useful in training
+about fundamental aspects of the design and development of CSE software as
+well as provide examples of continuous integration testing including
+demonstrating the use of ECP-CI resources.
+
+Each implementation we gather into this repository will have to conform
+to a *minimal* set of requirements including providing a fully self-contained
+build, accommodating a common set of command-line arguments, producing ascii
+text files for results output and providing time and space performance data via 
+tools such as [PAPI](https://icl.utk.edu/papi/) or [Caliper](https://software.llnl.gov/Caliper/).
+
+Our goal in collecting such examples isn't so much to demonstrate the use of
+different programming languages as it is to demonstrate different approaches
+to computation and execution paradigms as well as provide instructive insights
+by comparing of time, space and scaling performance of various approaches.
```

Comment:
We do not compare time, space, and scaling performance.

Reply:
As per conversation at DOE Data Day, the ultimate intention is to do so and its ok for this Blog article to announce that.